source/api_cc/include/DeepPot.h

```diff
@@ -480,39 +480,39 @@ class DeepPotModelDevi {
    **/
   double cutoff() const {
     assert(inited);
-    return rcut;
+    return dps[0].cutoff();
   };
   /**
    * @brief Get the number of types.
    * @return The number of types.
    **/
   int numb_types() const {
```

Comment:
It's a good practice to document the purpose and functionality of each method in a class. Consider adding brief comments for each method in the `DeepPotModelDevi` class. [medium]

Reply:
This should have been done...